### DISCUSSION TITLE
[NEXT-680] Help! Can't run the development server

### DISCUSSION THREAD

**Discussion by ricomuh** [vercel/next.js#46550] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

I can not run the development server either using npm run dev or yarn dev. I install new nex.js projects multiple times but it has the same result.

### Which area(s) of Next.js are affected? (leave empty if unsure)

App directory (appDir: true), SWC minifier (swcMinify: true), SWC transpilation

### Link to the code that reproduces this issue

https://github.com/ricomuh/my-app

### To Reproduce

When I run dev with the project that using app/ dir it's end in `info  - Using wasm build of next-swc`
![image](https://user-images.githubusercontent.com/76117803/221595666-784f2938-a74c-45d2-b2e8-54c970051d43.png)

But when I run the project that using `pages/` dir it's end in `ready - started server on 0.0.0.0:3000, url: http://localhost:3000` 
![image](https://user-images.githubusercontent.com/76117803/221595819-6b37687e-752d-467a-a23f-111c4471d0fe.png)

### Describe the Bug

 the terminal is ended when I run the dev server, I use git bash, tabby, and vscode terminal.

### Expected Behavior

it supposed to run normally.

### Which browser are you using? (if relevant)

Chrom 110.0.5481.177

### How are you deploying your application? (if relevant)

Vercel

<sub>[NEXT-680](https://linear.app/vercel/issue/NEXT-680/help-cant-run-the-development-server)</sub>

**Comment by balazsorban44** [comment]
Seems to be an OS-specific issue, I cannot reproduce it under Linux. :thinking:

The terminal output does not show any useful information though, so we might be swallowing some errors, will have to look more into this.

**Comment by ricomuh** [comment]
> Seems to be an OS-specific issue, I cannot reproduce it under Linux. :thinking:
> 
> The terminal output does not show any useful information though, so we might be swallowing some errors, will have to look more into this.

Well, I can run next projects that already installed before but not the projects that I installed since yesterday

**Comment by timneutkens** [comment]
Just ran the reproduction on codesandbox and couldn't reproduce either, you can view it here: https://codesandbox.io/p/github/ricomuh/my-app/main?file=%2FREADME.md

There's no much we can do to investigate this further without additional information like the output of `next info`.

**Comment by balazsorban44** [comment]
I tried and could not reproduce it on Windows either. 

An option is to try `next@canary` to see if it has been fixed, but for now, I'll move this to discussions.

If you still think it's a Next.js bug, please open a new issue with a proper reproduction, and system details (`next info`). :pray:

**Comment by icyJoseph** [comment]
@ricomuh What OS are you using? I've seen people with older MacOS systems have issues too. Could you be as specific as possible with your OS description? (Specifically after architecture, 32 bit? 64 bit?)

**Reply by ricomuh** [reply]
I use windows 10 64bit, now it's working when I uninstall my node.js and install it again. I think it's cache problem

**Reply by icyJoseph** [reply]
Aha, what Node version did you have before? Good that it work though!

**Reply by ricomuh** [reply]
I used node v16.14.0 before, now I installed the latest lts (18.14.2)

**Comment by asemali2210** [comment]
I also have the same issues I don't know what's the problem the terminal break when i try to run dev server

**Comment by sazy143** [comment]
Completely uninstalling node from my machine worked. Someone suggested a caching issue I am not sure if that was it. My Node was version 18.1 I think, and updated it to 18.15. Both of these ran issues with starting dev until I completely deleted and installed again.

**Comment by fresquer** [comment]
I had the same problem running Node 18.5 on my Windows system. I tried reinstalling Node and it worked

**Comment by icyJoseph** [comment]
I noticed that sometimes, it is possible to have a Node.js version that doesn't match your system's architecture, and that'll cause this error. 

If you are seeing this error, run: `node -p "process.arch"`, it should match your OS architecture (32/64 bits, etc), if it doesn't you must install the proper Node version for your OS.

I guess this is why, re-installing Node, "works" for most people.

**Reply by asemali2210** [reply]
Thank you for your reply.. it works may god make you happy

**Comment by popuguytheparrot** [comment]
same issue on 13.2.4 node 18.14.2
<img width="679" alt="Снимок экрана 2023-03-15 в 14 50 11" src="https://user-images.githubusercontent.com/19804652/225300855-4884ec13-3fdc-4dde-b584-8f91846f43c5.png">

**Reply by asemali2210** [reply]
I just uninstall my node.js and download the current version. 
Idk what's the main problem but this solve it

**Reply by icyJoseph** [reply]
As mentioned on my comment above, please run `node -p "process.arch"`, directly on your console, it should match your system's architecture.

**Reply by popuguytheparrot** [reply]
x64

**Reply by popuguytheparrot** [reply]
i use nvm for reinstall on mac

**Comment by Usama-Tariq** [comment]
If using windows, follow [this guide from Microsoft](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nextjs-on-wsl#prerequisites)
